test/csuite/timestamp_abort/main.c

```diff
@@ -412,6 +683,48 @@ thread_ckpt_run(void *arg)
             testutil_assert_errno(fclose(fp) == 0);
         }
     }
+
+    /* NOTREACHED */
+}
+
+/*
+ * thread_backup_run --
+ *     Runner function for the backup thread.
+ */
+static WT_THREAD_RET
+thread_backup_run(void *arg)
+{
+    THREAD_DATA *td;
+    WT_SESSION *session;
+    uint32_t last_backup, sleep_time, u;
+    int i;
+
+    td = (THREAD_DATA *)arg;
+    last_backup = 0;
+
+    testutil_check(td->conn->open_session(td->conn, NULL, NULL, &session));
+
+    /*
+     * Create backups until we get killed.
+     */
+    for (i = 1;; ++i) {
+        sleep_time = __wt_random(&td->extra_rnd) % MAX_CKPT_INVL;
```

Comment:
I thought about whether we wanted an independent MAX_BACKUP_INVL. But I think this is okay.